### PR TITLE
[SD-5135] Adds an email formatter, that uses jinja templates

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: superdesk/templates/email_article_body.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<html>
+<body>
+<h1>{{ article.place[0].name }}:&nbsp;{{ article.headline | safe }}</h1>
```
Comment (mdhaman) [inline superdesk/templates/email_article_body.html]: place is optional.

**Comment by petrjasek** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**